pkg/icingadb/ha.go

```diff
@@ -337,6 +338,22 @@ func (h *HA) realize(ctx context.Context, s *icingaredisv1.IcingaStatus, t *type
 					log("Can't update or insert instance. Retrying", zap.Error(err), zap.Uint64("retry count", attempt))
 				}
 			},
+			OnSuccess: func(elapsed time.Duration, attempt uint64, lastErr error) {
+				if attempt > 0 {
+					log := h.logger.Debugw
+
+					if attempt > 3 {
+						// We log errors with severity info starting from the fourth attempt,
```

Comment:
```suggestion
						// We log errors with severity info starting from the fifth attempt,
```

`attempt` is zero based. Otherwise, this might be the fourth retry.

Reply:
The first line is about errors logs above where we log with severity info starting from the fourth attempt. Then attempt is increased and the success handler has to account that.

Reply:
And yes, it is based on zero. But when it comes to logging and writing, we start with the first attempt.